S2VX.Desktop/Program.cs

```diff
@@ -1,6 +1,10 @@
 ﻿using osu.Framework;
 using S2VX.Game;
+using System;
 
+// Yeah, so I'm not sure if we need to be compliant. I don't think it matters
+// unless we're aiming to target multiple .NET languages?
+[assembly: CLSCompliant(false)]
```

Comment:
What are we doing that is not compliant?

Reply:
Ok, so this stuff's a bit confusing to me, but here's my understanding. There's the [Common Language Specification](https://docs.microsoft.com/en-us/dotnet/standard/language-independence-and-language-independent-components) that dictates rules that your code must conform to if you want that code to be able to run with different .NET languages. Since we aren't really exposing S2VX as a library or for other .NET languages besides C#, I don't think being CLS Compliant really matter to us.

But let's say we want to entertain the thought of being CLS Compliant. If you set `[assembly: CLSCompliant(true)]` in the and try to build, you'll find a lot of new CLS Compliant errors pop up, and that's actually because osu! framework itself is not marked as CLS Compliant. So now you'll have to go through everywhere that references osu! framework types and mark them with a `[CLSCompliant(false)]` attribute. At that point, I'd say that's too much busywork, and if the whole framework we're relying on isn't CLS Compliant, I don't think we can call ourselves compliant either.

Reply:
Ah ok, so it's osu! framework that's not compliant.